data-prepper-core/src/integrationTest/java/com/amazon/dataprepper/integration/EndToEndRawSpanTest.java

```diff
@@ -25,49 +24,70 @@
 import org.junit.Test;
 
 import java.io.IOException;
-import java.io.UnsupportedEncodingException;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.Random;
-import java.util.UUID;
 import java.util.concurrent.TimeUnit;
 
 import static org.awaitility.Awaitility.await;
 
 public class EndToEndRawSpanTest {
-
-    private static final Random RANDOM = new Random();
-    private static final List<Span.SpanKind> SPAN_KINDS =
-            Arrays.asList(Span.SpanKind.SPAN_KIND_CLIENT, Span.SpanKind.SPAN_KIND_CONSUMER, Span.SpanKind.SPAN_KIND_INTERNAL, Span.SpanKind.SPAN_KIND_PRODUCER, Span.SpanKind.SPAN_KIND_SERVER);
+    private static final int DATA_PREPPER_PORT_1 = 21890;
+    private static final int DATA_PREPPER_PORT_2 = 21891;
+
+    private static final String TEST_TRACEID_1 = "ABC";
+    private static final String TEST_TRACEID_2 = "CBA";
+    private static final Map<String, String> TEST_TRACEID_TO_TRACE_GROUP = new HashMap<String, String>() {{
+       put(Hex.toHexString(TEST_TRACEID_1.getBytes()), EndToEndTestData.DATA_100.name);
+        put(Hex.toHexString(TEST_TRACEID_2.getBytes()), EndToEndTestData.DATA_101.name);
+    }};
+    private static final List<EndToEndTestData> TEST_DATA_SET_1_WITH_ROOT_SPAN = Arrays.asList(
+            EndToEndTestData.DATA_100, EndToEndTestData.DATA_200, EndToEndTestData.DATA_300,
+            EndToEndTestData.DATA_400, EndToEndTestData.DATA_500, EndToEndTestData.DATA_600);
```

Comment:
Suggest making the test data names less arbitrary, it's difficult to understand what's going on unless we have the TestData.java class pulled up in another window.
```
EndToEndTestData.SPAN_1_SERVICE_A
or
EndToEndTestData.TRACE_1_SPAN_1_SERVICE_A
or
EndToEndTestData.TRACE_1_SPAN_1_SERVICE_A_CLIENT
```
Etc. whatever other fields would be useful to know in the name. Could even have something like `_SPAN_ROOT_` to indicate the parent span.